leptonai/templates/nemo_by_lepton/nemo.py

```diff
@@ -0,0 +1,34 @@
+import tempfile
+import os
+from leptonai.photon import Photon
+from fastapi import UploadFile, File
+
+class Transcriber(Photon):
+    # The init method implements any custom initialization logic we need.
+    requirement_dependency = ["Cython", "packaging", "nemo_toolkit[all]", "huggingface-hub==0.23.2", "numpy==1.26.3"]
+    system_dependency = ["libsndfile1", "ffmpeg"]
+
+    def init(self):
+        from nemo.collections.asr.models import EncDecMultiTaskModel
+        self.asr_model = EncDecMultiTaskModel.from_pretrained('nvidia/canary-1b')
```

Comment:
as a convention, we add an environment variable (and `deployment_template`) to let user specify underlying model  (with a good default)